packages/tools/devtools/devtools-core/api-extractor-esm.json

```diff
@@ -1,5 +1,4 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "../../../../common/build/build-common/api-extractor-base-esm.json",
-	"mainEntryPointFilePath": "<projectFolder>/lib/index.d.ts"
+	"extends": "../../../../common/build/build-common/api-extractor-base-esm.json"
```

Comment:
This is not the correct file to extend from. We should only extend from api-extractor-base.esm.primary.json or api-extractor-base.esm.secondary.json (expect in special cases).

Reply:
Hmm...  are you sure?  All other packages seems to be extending '.esm.json', which is just a thunk that includes .secondary.json:

```
{
	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
	"extends": "./api-extractor-base.esm.secondary.json"
}
```

Reply:
Right. Plain .esm came first but is was not what was needed for an ESM first package. So we then introduced .esm.primary/secondary for clarity.
But we did not go back and clean up all of the plain .esm uses. Lower priority (+I have been pushing for esm priority :) )